lib/nimble_template/addons/github.ex

```diff
@@ -84,12 +83,12 @@ defmodule NimbleTemplate.Addons.Github do
   end
 
   @impl true
-  def do_apply(%Project{} = project, opts) when not opts.github_action_deploy_heroku do
+  def do_apply(%Project{mix_project?: true} = project, %{github_action_deploy_heroku: true}) do
```

Comment:
I'm not sure if this is possible as on mix project we don't ask for Heroku deployment setup, right? 🤔

Reply:
That is a good question Ros, let me check the Mix template, and the Github Addon a bit.

Reply:
adjusted in https://github.com/nimblehq/elixir-templates/pull/152/commits/0f9b3b719a6e7a64d3054956ad12959845b7cbb0 @rosle